tests/e2e/backup_restore_suite_test.go

```diff
@@ -154,8 +154,15 @@ func runBackupAndRestore(brCase BackupRestoreCase, expectedErr error, updateLast
 	// wait for backup to not be running
 	Eventually(IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*20, time.Second*10).Should(BeTrue())
 	// TODO only log on fail?
-	GinkgoWriter.Println(DescribeBackup(veleroClientForSuiteRun, csiClientForSuiteRun, dpaCR.Client, backup))
-	Expect(BackupErrorLogs(kubernetesClientForSuiteRun, dpaCR.Client, backup)).To(Equal([]string{}))
+	describeBackup := DescribeBackup(veleroClientForSuiteRun, csiClientForSuiteRun, dpaCR.Client, backup)
+	GinkgoWriter.Println(describeBackup)
+
+	backupLogs := BackupLogs(kubernetesClientForSuiteRun, dpaCR.Client, backup)
+	backupErrorLogs := BackupErrorLogs(kubernetesClientForSuiteRun, dpaCR.Client, backup)
+	accumulatedTestLogs = append(accumulatedTestLogs, describeBackup)
+	accumulatedTestLogs = append(accumulatedTestLogs, backupLogs)
```

Comment:
can't we append all items at once? and why not also append ErrorLogs?

Reply:
Yes we can, there is `...` option, however I prefer to be explicit here:

```
accumulatedTestLogs = append(accumulatedTestLogs, describeBackup, backupLogs...)
```